### PR TITLE
feat: move signandsubmit

### PR DESCRIPTION
**Pull request by LeonFLK**
## fixes KILTProtocol/ticket#1247

Moves `signAndSubmitTx` as exported function to `BlockchainUtils`
added additional instruction for step by step signing and submitting in getting started guide
removed `SubscriptionPromise` from direct sdk-js package export, this is accessible in `ChainHelpers`

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [x] I have documented the changes (where applicable)

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -168,9 +168,16 @@ const identity = Kilt.Identity.buildFromMnemonic(
...
+
+// either sign and send in one step
 await Kilt.BlockchainUtils.submitSignedTx(tx, {
   resolveOn: Kilt.BlockchainUtils.IS_IN_BLOCK,
 })
+
+// or step by step
+const chain = Kilt.connect()
+const signed = chain.signTx(identity, tx)
+await chain.submitSignedTxWithReSigns(tx)
```
Comment (tjwelde) [inline docs/getting-started.md:180]: wouldn't this need an identity?

File: docs/getting-started.md [hunk 2]
```diff
@@ -168,9 +168,16 @@ const identity = Kilt.Identity.buildFromMnemonic(
   { signingKeyPairType: 'ed25519' }
 )
 const tx = await ctype.store(identity)
```
Comment (tjwelde) [inline docs/getting-started.md:170]: this doesn't take an identity anymore.

File: docs/getting-started.md [hunk 3]
```diff
@@ -168,9 +168,16 @@ const identity = Kilt.Identity.buildFromMnemonic(
...
+
+// either sign and send in one step
 await Kilt.BlockchainUtils.submitSignedTx(tx, {
```
Comment (tjwelde) [inline docs/getting-started.md:173]: This should be the same as in the getting-started.ts file

File: packages/sdk-js/src/index.ts [hunk 4]
```diff
@@ -25,7 +22,6 @@ export {
   Attester,
   Verifier,
   BlockchainUtils,
-  SubscriptionPromise,
```
Comment (tjwelde) [inline packages/sdk-js/src/index.ts:28]: Is this not needed anymore?

**Review by tjwelde — CHANGES_REQUESTED** [review]

File: packages/sdk-js/src/index.ts (hunk 4 above)
Comment (LeonFLK) [reply packages/sdk-js/src/index.ts:28]: Nope wasn't really needed in the first place, the before required interface lives in types.

**Comment by LeonFLK** [comment]
I switched to vanilla submitSignedTx for the step by step process.

**Review by tjwelde — APPROVED** [review]
LGTM

**Pull request merged**